core/src/test/java/org/mskcc/oncotree/utils/TumorTypesUtilTest.java

```diff
@@ -102,6 +109,29 @@ public void testFullOncoTreeCodeListByCode() throws Exception {
                     failureReport.append(makeMismatchMessage(oncoTreeCode, "name", returnedName, expectedName));
                     failureCount = failureCount + 1;
                 }
+                MainType expectedMainType = expectedTumorType.getMainType();
+                MainType returnedMainType = returnedTumorType.getMainType();
+                if (expectedMainType == null || returnedMainType == null) {
```

Comment:
Do you care if they are both null? I guess do you need this outer if statement here? That last else statement could just go after these two checks if one is null and not the other. Not sure if it matters though, performance could be better this way if we don't expect it to happen often (test passes).